.github/workflows/homebrew-core-head.yml

```diff
@@ -1,9 +1,12 @@
-# Verify Homebrew Core Formula Works
+# Cron to verify that the Homebrew Core Formula Works.
+# This formula is created by ???
```

Comment:
This formula is created by our release process with the PR to `homebrew/homebrew-core`. What this workflow does is uses the latest version of the formula at that repo, but `develop` branch source code from our PR. This serves two purposes:
- Verifies that our changes don't break brew
- Gives us time before release to fix these issues and adjust our homebrew formula if needed.